packages/nx/src/nx-cloud/generators/connect-to-nx-cloud/connect-to-nx-cloud.ts

```diff
@@ -72,18 +73,32 @@ async function createNxCloudWorkspace(
   return response.data;
 }
 
-function printSuccessMessage(url: string) {
+function printSuccessMessage(url: string, token: string) {
   let origin = 'https://nx.app';
   try {
     origin = new URL(url).origin;
   } catch (e) {}
 
+  const githubSlug = getGithubSlugOrNull();
+  const apiUrl = removeTrailingSlash(
+    process.env.NX_CLOUD_API || process.env.NRWL_API || `https://cloud.nx.app`
+  );
+  let connectCloudUrl;
+  if (githubSlug) {
+    connectCloudUrl = `${apiUrl}/setup/connect-workspace/vcs?provider=GITHUB&selectedRepositoryName=${encodeURIComponent(
+      githubSlug
+    )}`;
+  } else {
+    connectCloudUrl = `https://cloud.nx.app/setup/connect-workspace/manual?accessToken=${token}`;
+  }
   output.note({
     title: `Your Nx Cloud workspace is public`,
```

Comment:
We've received a lot of feedback that this message is kinda scary, and users aren't sure if they should be concerned: `Your Nx Cloud workspace is public`. 

Do we want to reword it while we're in here? Something like:

```
Your Nx Cloud workspace is ready

To claim it, connect it to your Nx Cloud account:
- Push your changes
- Connect your workspace here ${connectCloudUrl}
```